src/app/src/test/kotlin/org/imperial/mrc/hint/integration/TranslationTests.kt

```diff
@@ -24,6 +24,6 @@ class TranslationTests : SecureIntegrationTests() {
         assertError(responseEntity,
                 HttpStatus.BAD_REQUEST,
                 "INVALID_FILE",
-                "Le fichier doit être de type PJNZ, zip, mais il est du type geojson,geojson,geojson.")
```

Comment:
In the course of adding the adr flag to hintr, I tracked down and fixed this bug with the validation error messages